<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QListWidget" name="songsList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>281</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>15</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 25 15pt &quot;Ubuntu&quot;;</string>
   </property>
  </widget>
  <widget class="QSlider" name="processSlider">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>420</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {
  border: 0px solid #bbb;
}
 
QSlider::sub-page:horizontal {
  background: rgb(26,176,153);
  border-radius: 0px;
  margin-top: 8px;
  margin-bottom: 9px;
}

QSlider::add-page:horizontal {
  background:  rgb(136, 138, 133);
  border: 0px solid #777; 
  border-radius: 2px; 
  margin-top: 8px; 
  margin-bottom: 9px; 
}
 
QSlider::handle:horizontal { 
  background: rgb(26,176,153); 
  border: 1px solid rgba(102,102,102,102); 
  width: 8px; 
  height: 8px; 
  border-radius: 5px; 
  margin-top: 2px; 
  margin-bottom: 2px; 
}
 
QSlider::handle:horizontal:hover { 
  background: rgb(26,176,153); 
  border: 1px solid rgba(102,102,102,102); 
  border-radius: 5px; 
}
 
QSlider::sub-page:horizontal:disabled { 
  background: #bbb; 
  border-color: #999; 
}
 
QSlider::add-page:horizontal:disabled { 
  background: #eee; 
  border-color: #999; 
}
 
QSlider::handle:horizontal:disabled { 
  background: #eee; 
  border: 1px solid #aaa; 
  border-radius: 4px; 
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="totalTimeLabel">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>410</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 25 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>0:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addSongsButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(135, 0, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(135, 0, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="delectButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(135, 0, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="presentTimeLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>410</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 25 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>0:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;Source Code Pro&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(26, 176, 153);
font: 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/musicstop.jpg)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="previousButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>350</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="playPauseButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>350</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>350</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_volume">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>18</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/音乐播放器_04.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="volumeButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>63</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {
  border: 0px solid #bbb;
}

QSlider::sub-page:horizontal {
  background:  rgb(26,176,153);
  border-radius: 0px;
  margin-top: 8px;
  margin-bottom: 9px;
}
 
QSlider::add-page:horizontal {
  background:  rgb(136, 138, 133);
  border: 0px solid #777; 
  border-radius: 2px; 
  margin-top: 8px; 
  margin-bottom: 9px; 
}
 
QSlider::handle:horizontal { 
  background: rgb(255,255,255); 
  border: 1px solid rgba(102,102,102,102); 
  width: 8px; 
  height: 8px; 
  border-radius: 5px; 
  margin-top: 2px; 
  margin-bottom: 2px; 
}
 
QSlider::handle:horizontal:hover { 
  background: rgb(26,176,153); 
  border: 1px solid rgba(102,102,102,102); 
  border-radius: 5px; 
}
 
QSlider::sub-page:horizontal:disabled { 
  background: #bbb; 
  border-color: #999; 
}
 
QSlider::add-page:horizontal:disabled { 
  background: #eee; 
  border-color: #999; 
}
 
QSlider::handle:horizontal:disabled { 
  background: #eee; 
  border: 1px solid #aaa; 
  border-radius: 4px; 
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 25 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>songsList</zorder>
  <zorder>processSlider</zorder>
  <zorder>totalTimeLabel</zorder>
  <zorder>presentTimeLabel</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>addSongsButton</zorder>
  <zorder>clearButton</zorder>
  <zorder>previousButton</zorder>
  <zorder>nextButton</zorder>
  <zorder>label_volume</zorder>
  <zorder>volumeSlider</zorder>
  <zorder>label_3</zorder>
  <zorder>playPauseButton</zorder>
  <zorder>volumeButton</zorder>
  <zorder>delectButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
